src/client/components/pages/statistics.tsx

```diff
@@ -19,23 +19,28 @@
 import * as bootstrap from 'react-bootstrap';
 import * as utilsHelper from '../../helpers/utils';
 
-import PropTypes from 'prop-types';
+import {_EditorType} from '../../../types';
```

Comment:
And with the renaming below:
```suggestion
import {type Editor} from '../../../types';
```